tests/unit/js/test.subjects.js

```diff
@@ -0,0 +1,74 @@
+const sinon = require('sinon');
+const $ = require('../mockjQuery');
+window.$ = $;
+const subject = require('../../../openlibrary/plugins/openlibrary/js/subjects');
+
+beforeEach(() => {
+    window.$ = $;
```

Comment:
See my comment below. There is also no `window` object in a headless test environment. If needed you'll need to set window to something!